[2022-06-26 08:46:27,414] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-26T08:46:25.582563+00:00 [queued]>
[2022-06-26 08:46:27,430] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-26T08:46:25.582563+00:00 [queued]>
[2022-06-26 08:46:27,431] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-06-26 08:46:27,432] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-06-26 08:46:27,433] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-06-26 08:46:27,440] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): postgreSQL_getTable> on 2022-06-26T08:46:25.582563+00:00
[2022-06-26 08:46:27,447] {standard_task_runner.py:52} INFO - Started process 9693 to run task
[2022-06-26 08:46:27,452] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'SDG_Enrique_V0', 'postgreSQL_getTable', '2022-06-26T08:46:25.582563+00:00', '--job-id', '184', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/company_forecasting.py', '--cfg-path', '/tmp/tmp3lt2pcb2', '--error-file', '/tmp/tmpijeum_mo']
[2022-06-26 08:46:27,455] {standard_task_runner.py:77} INFO - Job 184: Subtask postgreSQL_getTable
[2022-06-26 08:46:27,501] {logging_mixin.py:104} INFO - Running <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-26T08:46:25.582563+00:00 [running]> on host 5e8dc4b25004
[2022-06-26 08:46:27,542] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=enrique-liebana@outlook.com
AIRFLOW_CTX_DAG_OWNER=Enrique
AIRFLOW_CTX_DAG_ID=SDG_Enrique_V0
AIRFLOW_CTX_TASK_ID=postgreSQL_getTable
AIRFLOW_CTX_EXECUTION_DATE=2022-06-26T08:46:25.582563+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-26T08:46:25.582563+00:00
[2022-06-26 08:46:27,553] {base.py:74} INFO - Using connection to: id: postgres_conection. Host: host.docker.internal, Port: 5432, Schema: data_company, Login: airflow, Password: XXXXXXXX, extra: None
[2022-06-26 08:46:40,275] {company_forecasting.py:82} INFO - Backup Company data in : /raw_data/company.csv
[2022-06-26 08:46:40,277] {python.py:118} INFO - Done. Returned value was: None
[2022-06-26 08:46:40,292] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=SDG_Enrique_V0, task_id=postgreSQL_getTable, execution_date=20220626T084625, start_date=20220626T084627, end_date=20220626T084640
[2022-06-26 08:46:40,325] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-26 08:46:40,372] {local_task_job.py:146} INFO - Task exited with return code 0
